modules/mono/glue/GodotSharp/GodotSharp/Compat.cs

```diff
@@ -8,6 +8,7 @@
 namespace Godot;
 
 #pragma warning disable CS1734 // XML comment on 'X' has a paramref tag for 'Y', but there is no parameter by that name.
+#pragma warning disable IDE0040 // Add accessibility modifiers. This is currently disabled because of https://github.com/dotnet/roslyn/issues/52904
```

Comment:
```suggestion
#pragma warning disable IDE0040 // Add accessibility modifiers.
    // TODO: This is currently disabled because of https://github.com/dotnet/roslyn/issues/52904
```